qemu/tests/hpt_huge_page_negative.py

```diff
@@ -0,0 +1,32 @@
+import logging
+import re
+
+from virttest import error_context
+from virttest import virt_vm
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    hpt guest's negative test:
+    1) Boot vm with incorrect hpt_huge_page value options
+    2) Check if can get the expected qemu output
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    vm = env.get_vm(params["main_vm"])
+    negative_type = params.get('negative_type')
+    error_msg = params.get('error_msg', '')
+    try:
+        vm.create(params=params)
+        output = vm.process.get_output()
+    except virt_vm.VMCreateError as e:
```

Comment:
When the guest is successfully launched, should be `test.error/fail`?

Reply:
Updated

Reply:
Can be easier with:
```python
    try:
        vm.create(params=params)
    except virt_vm.VMCreateError as e:
        o = e.output
    else:
        test.fail("Test failed since vm shouldn't be launched.")
```

Reply:
Updated